modules/currency.js

```diff
@@ -7,20 +7,22 @@ import {getHook} from '../src/hook.js';
 import {defer} from '../src/utils/promise.js';
 import {registerOrtbProcessor, REQUEST} from '../src/pbjsORTB.js';
 import {timedBidResponseHook} from '../src/utils/perfMetrics.js';
+import * as events from '../src/events.js';
```

Comment:
IS there any reason / benefit to change this to only import the `on` and `off` functions of `events.js` since those are the only bits used I think?